### PR TITLE
Rosenheim

### PR DESCRIPTION
**Pull request by AugustQu**

### CODE REVIEW

**Comment by AugustQu** [comment]
I receive this error:
`TypeError: the JSON object must be str, not 'bytes'`

Here on my PC everything works fine.

What can I do to solve this?

PS: I'm using Python 3.6.8

File: park_api/cities/Rosenheim.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    }
+
+    # load the JSON-file:
+    urlHD = 'https://www.rosenheim.de/index.php?eID=jwParkingGetParkings'
+    headerHD={'Accept': 'application/json; charset=utf-8', 
+               'User-Agent': 'ParkAPI v%s - Info: %s' %(env.SERVER_VERSION, env.SOURCE_REPOSITORY) }
+    req = urllib.request.Request(url=urlHD, headers=headerHD)
+    webURL = urllib.request.urlopen(req)
+    data=webURL.read()
+    dataJSON=json.loads(data)
```
Comment (jklmnn) [inline park_api/cities/Rosenheim.py]: Why do you call urllib manually instead of using the argument `html` that provides exactly the same data?

Comment (kiliankoe) [reply park_api/cities/Rosenheim.py]: Because of the headers, which can't be specified otherwise.

Comment (AugustQu) [reply park_api/cities/Rosenheim.py]: sorry for beeing so late with a reply.

I don't know how to read this via the usual way. I have to specify the type JSON to get the data.

PS: I don't like this approach either.

Comment (AugustQu) [reply park_api/cities/Rosenheim.py]: what can I do to fix this error:
`TypeError: the JSON object must be str, not 'bytes'`

here everything works fine.

Comment (AugustQu) [reply park_api/cities/Rosenheim.py]: OK, seems to work now.

Comment (jklmnn) [reply park_api/cities/Rosenheim.py]: @kiliankoe I see. I thought those were only the headers we're already sending but that's not the case for the `Accept` header.

**Pull request closed without merging**